src/resources/views/todo/edit.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.base')
+@section('content')
+<div class="row justify-content-center">
+  <div class="col-md-8">
+    <div class="card">
+      <div class="card-header">ToDo編集</div>
+      <div class="card-body">
+        <form method="POST" action="{{route('todo.update',$todo->id)}}">
```

Comment:
[nits]
前後にスペース空けるで統一しましょうか！
```suggestion
        <form method="POST" action="{{ route('todo.update',$todo->id) }}">
```